euler_algs.py

```diff
@@ -50,3 +51,62 @@ def isSquare(n):
             return False
         seen.add(x)
     return True
+
+
+def isPandigital(n, digits='123456789'):
+    """
+    :param n: The number to check if it contains all of the numbers in :param digits once
+    :param digits: a string containing the numbers to check for pandigital numbers
+    :return: Boolean
+    """
+    t = digits
+    x = str(n)
+    for i in x:
+        digits = digits.replace(i, '')
+    return len(x) == len(t) and len(digits) == 0
+
+
+def isAnagram(x, y):
```

Comment:
This appears to be a less generalized case of `arePermutations` in euler_counting. Unless you think it's faster, I don't think we'll need it.